File: CMS/src/app/shared/CompanyFields.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { CompanyFields } from 'src/app/shared/CompanyFields.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class CompanyFieldsService extends BaseService{
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "CompanyFields";
    }
}

File: Service/Implement/CompanyFieldsService.cs
namespace Service.Implement
{
    public class CompanyFieldsService : BaseService<CompanyFields, ICompanyFieldsRepository>
    , ICompanyFieldsService
    {
    private readonly ICompanyFieldsRepository _CompanyFieldsRepository;
    public CompanyFieldsService(ICompanyFieldsRepository CompanyFieldsRepository) : base(CompanyFieldsRepository)
    {
    _CompanyFieldsRepository = CompanyFieldsRepository;
    }
    }
    }

